### DISCUSSION TITLE
调用 search restFul 接口错误

### DISCUSSION THREAD

**Discussion by revert-yang** [milvus-io/milvus#26162] [Q&A and General discussion]
调用${MILVUS_HOST}:${MILVUS_PORT}/v1/vector/search接口，返回如下错误：

checkIfLoaded failed when search, collection:id_89b9ee02_0189_b9ee02b3_4028888f_0000_720, partitions:[], err = GetCollectionInfo failed, collection = id_89b9ee02_0189_b9ee02b3_4028888f_0000_720, err = collection 443321495447088295 has not been loaded to memory or load failed

**Comment by yhmo** [comment]
“collection 443321495447088295 has not been loaded to memory or load failed”

Milvus requires the collection to be loaded into memory before searching. You need to call `${MILVUS_PORT}/v1/collection/load`

**Comment by revert-yang** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
